README.md

```diff
@@ -41,6 +41,13 @@ FingerprintJS Pro Svelte SDK is an easy way to integrate [Fingerprint Pro](https
 
 This package works with Fingerprint Pro, it is not compatible with open-source FingerprintJS. See our documentation to learn more about the [difference between Fingerprint Pro and the open-source FingerprintJS](https://dev.fingerprint.com/docs/pro-vs-open-source).
 
+## Requirements
+
+- Svelte 3.53.1 or higher
```

Comment:
Is there any reason for so precise version?

Reply:
I used the version definied in `peerDependencies` for it (actually, I just changed it because it should be 3.30.1 :smile: